### DISCUSSION TITLE
Implement DD mechanics

### DISCUSSION THREAD

**Discussion by jjppof** [jjppof/goldensun_html5#210] [Ideas]
- Support transformation-based classes
- Allow diagonal, radius-based field psynergy targeting (L/R to switch between targets)
- Allow Djinni to have individual sprites
- Support multiple unleashes per weapon
- Support more than 4 elements

**Comment by Salanewt** [comment]
Don't forget Ghoul Ball's mechanic of putting djinn into "permanent recovery" that only ends when the enemy dies or when battle ends.
